dockers/docker-database/multi_database_config.json.j2

```diff
@@ -0,0 +1,192 @@
+{% set include_remote_db = (REMOTE_DB_IP is defined and REMOTE_DB_PORT is defined) %}
+{
+    "INSTANCES": {
+        "redis":{
+            "hostname" : "{{HOST_IP}}",
+            "port" : {{REDIS_PORT}},
+            "unix_socket_path" : "/var/run/redis{{DEV}}/redis.sock",
+            "persistence_for_warm_boot" : "yes"
+            {% if DATABASE_TYPE is defined and DATABASE_TYPE != "" %}
+            ,"database_type": "{{DATABASE_TYPE}}"
+            {% endif %}
+        },
+        "redis1": {
+           "hostname" : "{{HOST_IP}}",
+            "port" : 6380,
```

Comment:
The port selection conflicts with both the chassis DB and Smart Switch DB service and must be changed. Port 6380 is used by the chassis Redis server, while ports 6381-6388 are assigned to the DPU databases.

Reply:
@oleksandrivantsiv where does chassis DB's port get defined?

This multi-db approach is enabled only on the platform which needs performance and controlled by compile flag. ENABLE_MULTIDB=y  The detail information could be found at https://lists.sonicfoundation.dev/g/sonic-wg-routing/wiki/37961

During the discussion in routing WG, we feel there is NO use case to enable it for chassis based platform for the following reasons.

1. The scale for chassis based approach is limited. MSFT uses it as T1, and Alibaba uses it as gateway. The benefit is less. 
2. Even chassis case really wants to use it. It may only wants to enable it for main switch asic, not for each DPU.  That means we need to create a new j2 for that case.  You may want to blindly create 5 redis instances for each DPU database. That causes too many redis instances in the system.

Reply:
Discussed with Guahan on this one. 6380 will be kept for Chassis DB. It is not easy to change chassis's port number. 

We use 6379, 6378, 6377, ...  a.k.a going down for new redis instances. 

@Pan-XT can you update your diff?

Reply:
Update redis1,reids2,redis3,redis4 port number to 6378,6377,6376,6375

Reply:
#closed